src/rustdoc/README.md

```diff
@@ -0,0 +1,20 @@
+# Rustdoc
+Rust's rustdoc team members are responsible for maintaining the Rustdoc tool, improving its performance
+and considering the stabilization of rustdoc features.
```

Comment:
We should mention code review here, coming from nixpkgs it wasn't immediately obvious how `r+` is linked to team membership.

This is explained in detail later but could probably use a passing mention here.

Reply:
Seems a bit weird to put it in the summary though.